projects/isy-angular-widgets/src/lib/security/security-guard.spec.ts

```diff
@@ -7,47 +7,57 @@ import {of} from 'rxjs';
 import SpyObj = jasmine.SpyObj;
 import {PermissionMaps} from './permission-maps';
 
-describe('SecurityGuard - without setting up roles and permissions', function() {
```

Comment:
Folgende Anpassungen bei dem Test fände ich gut:

- Jede Methode des Service in einem separaten Test abdecken (check Element, check Route sync, check Route async)
- Die Testdaten sollen nicht aus der Demo Anwendung kommen, m.E. sollte es keine Abhängigkeitsrichtung DemoApp --> Library geben.

Reply:
1. cyclic dependencies in security-service.spec.ts and security-guard.spec.ts was removed
2. security-service.spec.ts: 2x Tests -> 6x Tests